example-setup/scripts/install_generic.sh

```diff
@@ -59,7 +85,8 @@ EOF
 systemctl enable consul
 systemctl restart consul
 
-until consul join 192.168.2.11 192.168.2.12 192.168.2.13; do sleep 2; done
+# join configured in consul.json
+#until consul join 192.168.2.11 192.168.2.12 192.168.2.13; do sleep 2; done
```

Comment:
If we set the join beaviour inside the config file we can safely delete it here.